apps/condo/domains/acquiring/routes/paymentLinkRouter.js

```diff
@@ -119,6 +140,11 @@ class PaymentLinkRouter {
     }
 
     async handleRequest (req, res) {
+        const featureFlagsHeader = get(req, ['headers', 'feature-flags'])
+        if (featureFlagsHeader) {
+            this.context.req = { headers: { 'feature-flags': featureFlagsHeader } }
```

Comment:
This is still inappropriate to touch shared context. I suggest to construct a new object and pass it into  isFeatureEnabled. Maybe { req } will work?

Reply:
Refactored